_docs/v2.24/setup/install.md

```diff
@@ -77,7 +77,7 @@ Create a file named `main.tf` in an empty directory and copy the contents below
 ```hcl
 module "forseti" {
   source  = "terraform-google-modules/forseti/google"
-  version = "~> 5.0.0"
+  version = "~> 5.1.1"
```

Comment:
Change to "~>5.1" so that users can pick up micro versions (e.g. 5.1.1, 5.1.2) without having to change this variable.

Reply:
Done